packages/bundler-sdk-viem/src/BundlerAction.ts

```diff
@@ -1020,7 +1027,15 @@ export namespace BundlerAction {
         data: encodeFunctionData({
           abi: blueAbi,
           functionName: "setAuthorizationWithSig",
-          args: [authorization, { v: yParity + 27, r, s }],
+          args: [
+            {
+              ...authorization,
+              // Never permit any other address than the GeneralAdapter1 otherwise
+              // the signature can be extracted and used independently.
+              authorized: generalAdapter1,
```

Comment:
That becomes wrong

That's why I would rather throw here is unexpected value rather than silently overiding